### PR TITLE
created login page

### PR DESCRIPTION
**Pull request by akhilarjun3-sureify**
login-output-screenshot
login-output-screenshot2

### CODE REVIEW

**Review by harshavardhan-sureify — APPROVED** [review]

**Review by maniknr-sureify — APPROVED** [review]

File: src/components/LoginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  const [submitStatus, setSubmitStatus] = useState("");
+  const [seePassword, setSeePassword] = useState(false);
+  const sendDataToServer = async (data) => {
+    try {
+      const res = await fetch(login, {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify(data),
+      });
+      if (res.ok) {
```
Comment (abhishekdundigalla-sureify) [inline src/components/LoginPage.jsx:42]: better compare with response status

File: src/components/LoginPage.jsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        </Grid>
+        <Grid item xs={6} md={6} lg={4}>
+          <Paper
+            elevation={3}
+            sx={{ padding: "2px 15px", margin: "20px 0px", height: "100%" }}
+            align={"center"}
+          >
+            {submitStatus && (
+              <Alert severity="error">
+                {/* <Typography variant="p">{submitStatus}</Typography> */}
```
Comment (abhishekdundigalla-sureify) [inline src/components/LoginPage.jsx:126]: remove comments

File: src/components/LoginPage.jsx [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+  Paper,
+  Toolbar,
+  Typography,
+} from "@mui/material";
+
+import LockIcon from "@mui/icons-material/Lock";
+import React, { useState } from "react";
+import { theme } from "../themes/theme";
+import logo from "../images/logo.png";
+import profileImage from "../images/profileImage.png";
```
Comment (Madhukar-Sureify) [inline src/components/LoginPage.jsx:18]: can i know the location where the profile image and logo is stored?

Comment (akhilarjun3-sureify) [reply src/components/LoginPage.jsx:18]: In src folder ,created a folder called images in that i stored those images.

Comment (Madhukar-Sureify) [reply src/components/LoginPage.jsx:18]: its better to have assets in src then add image folder  add images and icons into that folder kindly update at corresponding places

Comment (akhilarjun3-sureify) [reply src/components/LoginPage.jsx:18]: ok

File: src/components/LoginPage.jsx (hunk 2 above)
Comment (akhilarjun3-sureify) [reply src/components/LoginPage.jsx]: yes

File: src/components/LoginPage.jsx (hunk 1 above)
Comment (akhilarjun3-sureify) [reply src/components/LoginPage.jsx]: yes

File: src/components/LoginPage.jsx [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+              </Button>
+              <Typography variant="p">
+                <Link to="/forgotpassword" style={{ textDecoration: "none" }}>
+                  Forgot password?
+                </Link>
+              </Typography>
+              <Typography variant="body2">
+                Dont have an account?{" "}
+                <Link to="/" style={{ textDecoration: "none" }}>
+                  Signup
```
[the remark this answers is not in the document]
Comment (akhilarjun3-sureify) [reply src/components/LoginPage.jsx:193]: changed

**Review by abhishekdundigalla-sureify — APPROVED** [review]

**Review by Madhukar-Sureify — APPROVED** [review]

**Pull request merged**